src/client/app/services/github.service.ts

```diff
@@ -95,7 +95,7 @@ export class GitHubService {
         if (this.profile == null) {
             return Observable.of([]);
         }
-        let url = `${this._baseUrl}/users/${this.profile.login}/gists`;
+        let url = `${this._baseUrl}/users/${this.profile.login}/gists?timestamp=${(new Date).getTime()}`;
```

Comment:
Discussed offline. Added optional parameter for get() function in 'app/services/request.ts' to use cache or not; if this is set to 'false', then the timestamp random number param is applied. The GitHub API seems to reject using 'Cache-Control' in the headers. Would welcome other suggestions.